app/components/autocomplete/at_mention/index.js

```diff
@@ -25,6 +25,15 @@ import {Permissions} from '@mm-redux/constants';
 import AtMention from './at_mention';
 
 function mapStateToProps(state, ownProps) {
+    if (ownProps.appsTakeOver) {
+        // Return empty values for the requiredfields.
+        return {
+            currentTeamId: '',
+            requestStatus: '',
+            theme: {},
```

Comment:
this one will cause the component to re-render every time mapStateToProps run as is always a different object